rfcs/tabs-component.md

```diff
@@ -0,0 +1,150 @@
+# Tabs Component
+
+## Usage
+
+### Basic usage
+
+```js
+import * as React from 'react';
+import {Tabs, TabPanel} from 'baseui/tabs';
+
+export default () => {
+  return (
+    <div>
+      <Tabs initialState={{activeKey: '1'}}>
+        <TabPanel key="1" title="Tab 1">Tab 1 Content</Tab>
```

Comment:
should we keep the basic usage absolute basic? so for example, leaving out the `key` prop, as that's optional?